tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNetCore/ActionResponseFilter.cs

```diff
@@ -0,0 +1,32 @@
+// <copyright file="ActionResponseFilter.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if !NETFRAMEWORK
+
+using Datadog.Trace.AppSec;
+using Datadog.Trace.AppSec.Coordinator;
+using Datadog.Trace.DuckTyping;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AspNetCore;
+
+internal class ActionResponseFilter : IActionFilter
```

Comment:
We've discussed in the past that we _shouldn't_ be referencing the AspNetCore assemblies directly, and instead using DuckType only 😕 That said, we're already referencing this lib directly, so maybe it's not a problem 🤷‍♂️ I don't think it's a deal breaker or anything, just something we should perhaps re-evaluate

Reply:
yes.. so the library was already referenced, so just took advantage of it but didnt add the reference 😅